WordPress/twentytwentyfive
# Add theme-utils

**Description**



Closes #4 

Copied from Automattic/themes since there are some recent changes that haven't made it into WordPress/community-themes yet. I had to make some modifications so that it can work on single-theme repos.

This does not include the modifications that `escape-patterns` would make to files or GitHub Automations to run validation on PRs.

Difference from WordPress/community-themes:

- Names of npm scripts
  - **`lint:theme`**: Instead of `patterns:escape` in community-themes
  - **`lint:patterns:fix`**: Instead of `schema:validate` in community-themes
- `escape-patterns` runs on all patterns instead of only modified files. (See `node theme-utils.mjs help escape-patterns` for instructions to run on only modified files.)

**Testing Instructions**






Don't forget to `npm install` as there are some new dev dependencies.

Run both commands
- `npm run lint:theme`
- `npm run lint:patterns:fix`

Verify issues/changes reported are correct.
